### PR TITLE
Update zh-TW

### PR DESCRIPTION
**Pull request by Neo1102**
improve apd_zh_TW.ts and add README.md for traditional chinese

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@
         <img src="https://img.shields.io/badge/license-GPLv3-yellow.svg"/>
     </a>
 </p>
-<p align="center">🌎 English | 🌏 <a href="/README-CN.md">简体中文</a></p>
+<p align="center">🌎 English </a>| 🌏 <a href="/README-TW.md">繁體中文</a> | 🌏 <a href="/README-CN.md">简体中文</a></p>
```
Comment (SpriteOvO) [inline README.md:24]:
```suggestion
<p align="center">🌎 English | 🌏 <a href="/README-CN.md">简体中文</a> | 🌏 <a href="/README-TW.md">繁體中文</a></p>
```

The first newly added `</a>` seems unnecessary, and according to the alphabetical order, `zh_TW` should be better placed after `zh_CN`.

File: README-CN.md [hunk 2]
```diff
@@ -21,7 +21,7 @@
         <img src="https://img.shields.io/badge/license-GPLv3-yellow.svg"/>
     </a>
 </p>
-<p align="center">🌎 <a href="/README.md">English</a> | 🌏 简体中文</p>
+<p align="center">🌎 <a href="/README.md">English</a> | 🌏 <a href="/README-TW.md">繁體中文</a> | 🌏 简体中文</p>
```
Comment (SpriteOvO) [inline README-CN.md:24]:
```suggestion
<p align="center">🌎 <a href="/README.md">English</a> | 🌏 简体中文 | 🌏 <a href="/README-TW.md">繁體中文</a></p>
```

File: README-TW.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        <img src="https://img.shields.io/github/downloads/SpriteOvO/AirPodsDesktop/total.svg"/>
+    </a>
+    <a href="https://github.com/SpriteOvO/AirPodsDesktop/compare">
+        <img src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg"/>
+    </a>
+    <a href="/LICENSE">
+        <img src="https://img.shields.io/badge/license-GPLv3-yellow.svg"/>
+    </a>
+</p>
+<p align="center">🌎 <a href="/README.md">English</a> | 🌏 繁體中文</a> | 🌏 <a href="/README-CN.md">简体中文</a></p>
```
Comment (SpriteOvO) [inline README-TW.md:24]:
```suggestion
<p align="center">🌎 <a href="/README.md">English</a> | 🌏 <a href="/README-CN.md">简体中文</a> | 🌏 繁體中文</p>
```

Comment (SpriteOvO) [reply README-TW.md:24]: Same, a unnecessary `</a>` and the order issue.

**Review by SpriteOvO — CHANGES_REQUESTED** [review]
Appreciate your contribution to translations! I just found some minor issues, you can continue to commit changes on your branch to fix them.

File: README-TW.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
+<h1 align="center">
+    <a href="https://github.com/SpriteOvO/AirPodsDesktop"><img src="/Source/Resource/Image/Icon.svg" alt="Icon" width="128"></a>
+    <br>
+    AirPodsDesktop
+</h1>
+<p align="center"> AirPods 桌面使用者體驗增進軟體</p>
```
Comment (SpriteOvO) [inline README-TW.md:6]:
```suggestion
<p align="center">AirPods 桌面使用者體驗增進軟體</p>
```

**Comment by Neo1102** [comment]
Hi
I have confirmed that your modifications have been applied to my branch. Do I need to open a new pull request, or is there any step I might have missed?

**Comment by SpriteOvO** [comment]
Oh, sorry. No more action is needed, I just forgot to process it yesterday. I will merge it today.

**Comment by SpriteOvO** [comment]
Thanks for your contributions again!

**Pull request merged**